04-gas/gas-section-one/gas-section-one.css

```diff
@@ -56,24 +58,46 @@
 
 @media (min-width: 600px) {
   .gas-section-one-copy-container {
+    align-content: center;
     row-gap: var(--copy-container-row-gap-desktop);
   }
 }
 
+.gas-section-one-title-container{
+}
+
+@media (min-width: 600px){
+  .gas-section-one-title-container{
+   
+  }
+}
+
 .gas-section-one-title {
+  display: flex;
+  flex-direction: column;
   text-align: center;
-  font-size: var(--title-font-size-mobile);
-  font-weight: var(--title-font-weight-mobile);
+  row-gap: var(--title-container-row-gap-mobile);
```

Comment:
change this to reflect global var name
if it is something specific to gas components then, something like:
`gas-title-row-gap-mobile` (??)